components/FeedbackView/ButtonGroup.test.tsx

```diff
@@ -0,0 +1,56 @@
+import { render, screen } from '@testing-library/react';
+import ButtonGroup from './ButtonGroup';
+
+it('Each button renders with the correct title and corresponding subtitle', () => {
```

Comment:
phrasing (nit): for tests it should read e.g. 

```suggestion
it('Renders each button with the correct title and corresponding subtitle', () => {
```

would read as `It renders each button with the correct title and corresponding subtitle`, instead of `It each button renders with the correct title and corresponding subtitle`